src/prognostic_equations/implicit/implicit_solver.jl

```diff
@@ -357,7 +357,7 @@ function update_implicit_equation_jacobian!(A, Y, p, dtγ, colidx)
     (; ᶜspecific, ᶠu³, ᶜK, ᶜts, ᶜp, ᶜΦ, ᶠgradᵥ_ᶜΦ, ᶜρ_ref, ᶜp_ref) = p
     (; ∂ᶜK_∂ᶜuₕ, ∂ᶜK_∂ᶠu₃, ᶠp_grad_matrix, ᶜadvection_matrix) = p
     (; ᶜdiffusion_h_matrix, ᶜdiffusion_u_matrix, params) = p
-    (; energy_upwinding, density_upwinding) = p.atmos.numerics
+    (; energy_upwinding) = p.atmos.numerics
```

Comment:
We also need to remove `energy_upwinding`, `tracer_upwinding`, and `precip_upwinding` from this function. They should all be set to `Val(:none)` here.